pkg/controller_workload/workload/workload_event.go

```diff
@@ -514,3 +521,29 @@ func handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse)
 	}
 	return err
 }
+
+func (svc *ServiceEvent) handleAuthorizationTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse) error {
+	auth := &security.Authorization{}
+
+	// update resource
+	for _, resource := range rsp.GetResources() {
+		if err := anypb.UnmarshalTo(resource.Resource, auth, proto.UnmarshalOptions{}); err != nil {
+			log.Errorf("unmarshal failed, err: %v", err)
+			continue
+		}
+		log.Debugf("handle auth xds, resource.name %s, auth %s", resource.GetName(), auth.String())
+		if err := svc.rbac.UpdatePolicy(auth); err != nil {
```

Comment:
This is not right, as auth is a pointer type, eventually we can get  same policy storred

Reply:
Should move L526 into the for loop

Reply:
done